src/screens/ContactScreen/index.js

```diff
@@ -212,11 +228,13 @@ const ContactScreen = ({navigation}) => {
         // onPress={handleSearch}
       />
 
-      <View>
-        {selectedUsers && (
-          <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
-        )}
-      </View>
+      {sourceScreen !== ANONYMOUS && (
+        <View>
+          {selectedUsers && (
+            <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+          )}
+        </View>
+      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The rendering of the ContactPreview component has been updated to handle different scenarios based on the source screen. If the source screen is not `ANONYMOUS`, the ContactPreview component is rendered; otherwise, it is not. This change enhances the flexibility of the component rendering process.
```diff
+ {sourceScreen !== ANONYMOUS && (
+   <View>
+     {selectedUsers && (
+       <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+     )}
+   </View>
+ )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->